### PR TITLE
Display relevant OAuth information on accounts page

### PR DESCRIPTION
**Pull request by anshhhhhhh**

### CODE REVIEW

File: client/src/route/auth/AccountTabs.tsx [hunk 1]
```diff
@@ -1,21 +1,80 @@
...
+  const auth = useAuth();
+  const groups: string[] = (auth.user?.profile.groups_direct as string[]) || [];
+
+  if (groups.length === 0) {
+    const accountTab: TabData[] = [
+      {
+        label: 'Profile',
+        body: (
+          <>
+            Profile - potentially visible to other users.
```
Comment (prasadtalasila) [inline client/src/route/auth/AccountTabs.tsx:18]: please remove these two lines

File: client/src/route/auth/AccountTabs.tsx [hunk 2]
```diff
@@ -1,21 +1,80 @@
...
+            <br></br>
+            <img
+              src={auth.user?.profile.picture}
+              style={{ height: '5%', width: '5%' }}
+            ></img>
+            Username is <b>{auth.user?.profile.preferred_username}</b>
+            <br></br>
+            See more details on{' '}
+            <a href={auth.user?.profile.profile}>SSO OAuth provider.</a>
+          </>
```
Comment (prasadtalasila) [inline client/src/route/auth/AccountTabs.tsx:28]: Please add the groups text and code here. The exact text is available in issue #478

File: client/yarn.lock
Comment (prasadtalasila) [inline client/yarn.lock:1]: There aren't any changes in the package.json dependencies. Is there any particular reason for updating this file?

File: client/src/route/auth/AccountTabs.tsx
Comment (prasadtalasila) [inline client/src/route/auth/AccountTabs.tsx:1]: Please add the unit and integration tests are required for this update.

**Review by prasadtalasila — COMMENTED** [review]
Broader comments:

1. There does not seem to be any change on the profile pages. Please check.
1. The codeclimate issues need to be resolved

File: client/src/route/auth/AccountTabs.tsx (hunk 2 above)
Comment (anshhhhhhh) [reply client/src/route/auth/AccountTabs.tsx:28]: Have added it in the conditional when the user is part of group/proups, this is tge the accounTabs constant content when he is not part of any of them

**Comment by prasadtalasila** [comment]
@anshhhhhhh you added the number of groups a user belongs to. Listing of these user groups is needed instead. Thanks.

**Comment by prasadtalasila** [comment]
@anshhhhhhh 
The Account.tsx and its test code has been updated in the latest commit on `feature/distributed-demo`. Please port your changes onto that branch. Thanks.

**Pull request closed without merging**